build.gradle

```diff
@@ -61,9 +65,10 @@ subprojects {
         compile group: 'com.google.code.findbugs', name: 'jsr305', version: '3.0.2'
         testCompile "com.hazelcast.jet:hazelcast-jet-core:" + jetVersion + ":tests"
         testCompile "com.hazelcast:hazelcast:" + hazelcastVersion + ":tests"
+        testCompile "com.hazelcast:hazelcast-enterprise:" + hazelcastVersion + ":tests"
```

Comment:
why does it use enterprise here?

Reply:
Because of the `TestKeyStoreUtil` used in tests